src/common/i18n/locales/pt-br.json

```diff
@@ -0,0 +1,3 @@
+{
+    
+}
```

Comment:
Podemos pensar em colocar o i18n em javascript, vale lembrar quando for discutir sobre isso. [LEMBRETE]